packages/gasket-plugin-config/test/plugin.test.js

```diff
@@ -57,6 +57,15 @@ describe('Plugin', () => {
       });
     });
 
+    it('uses a src directory if specified or default', () => {
```

Comment:
It looks like this concept of changing the config directory is pretty well tested. You can probably back out these changes.